modules/understanding-upgrade-channels.adoc

```diff
@@ -7,104 +7,100 @@
-Update stability depends on your channel. The presence of an update recommendation in the `candidate-{product-version}` channel does not imply that the update is supported. It means that no serious issues have been found with the update yet, but there might not be significant traffic through the update to suggest stability. The presence of an update recommendation in the `fast-{product-version}` or `stable-{product-version}` channels at any point is a declaration that the update is supported. While releases will never be removed from a channel, update recommendations that exhibit serious issues will be removed or made conditional from all channels. When an update recommendation is supported, it remains supported for the life of {product-version}, even if the update recommendation is later dropped or made conditional.
+[id="conditional-updates-overview_{context}"]
+== Update recommendation removals and Condtional Updates
+Red Hat monitors newly released versions and update paths associated with those versions both before and after they're added to supported channels. If a serious regression is identified Red Hat may remove affected update recommendations. When Red Hat chooses to remove update recommendations that action is taken in all relevant channels, simultaneously. The removal of recommended updates may happen either before or after updates have been promoted to supported channels.
 
-Red Hat will eventually provide supported update paths from any supported release in the `fast-{product-version}` or `stable-{product-version}` channels to the latest release in {product-version}.z, although there can be delays while safe paths away from troubled releases are constructed and verified.
-endif::openshift-origin[]
+In the event that Red Hat removes update recommendations from any supported release we will eventually provide a superseding update recommendation to a future version which corrects the regression. There may however be a delay while the defect is corrected, tested, and promoted to your selected channel.
+
+Beginning with {product-title} 4.10 whenever update recommendations are removed from supported channels they are replaced with Conditional Updates which declare one or more known risks. Each known risk may apply to all clusters or only clusters matching certain conditions such as having `Platform` set to `None`` or the CNI provider set to `OpenShiftSDN`. The Cluster Version Operator (CVO) continually evaluates known risks against the current cluster state. When no risks match, the update is recommended. You can update from the Administrator perspective on the web console, as well as the OpenShift CLI (`oc`).
+
+When an update is not recommended because a risk might apply, you can view the update from the OpenShift CLI (`oc`). If the cluster administrator evaluates the potential known risks and decides it is acceptable for the current cluster, the administrator can waive the safety guards and proceed to an update.
```

Comment:
The pre-existing `oc` focus here was because we landed this language before the console grew a UX for updating despite matching risks.  They [backported that UX in 4.10.16][1], and these docs are aimed at 4.12 and later, so we can probably either:

* Mention "the Administrator perspective on the web console" in this paragraph, or
* Drop all client references from both this and the previous paragraph, focusing on the workflow semantics, and leaving it implicit that both of our update clients support all of the discussed workflows.

[1]: https://bugzilla.redhat.com/show_bug.cgi?id=2087041#c5

Reply:
PTAL, I've done the bare minimum but we should probably take a card to work with docs to walk through the web console workflow, but apparently we're not allowed to use screenshots so that'll be fun.